src/app/setup-wizard/data-collection/on-site-assessment/assessment-nebs-form/assessment-nebs-form.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="d-flex w-100 justify-content-end pt-2 pb-2">
     <button class="me-2 btn btn-secondary btn-sm add-neb-btn" (click)="addNEB()">
-        <fa-icon [icon]="faPlus"></fa-icon> Add NEB
+        <fa-icon [icon]="faPlus"></fa-icon> Add Custom NEB
```

Comment:
could do the same thing with the naming in src\app\setup-wizard\data-collection\on-site-assessment\assessment-energy-opportunities-form\energy-opportunity-setup-form\energy-opportunity-setup-form.component.html

Reply:
There is an issue to consolidate these buttons (#179). Will handle that then.